remme/rest_api/token.py

```diff
@@ -23,7 +23,8 @@ def get(pub_key_user):
     address = client.make_address_from_data(pub_key_user)
     print('Reading from address: {}'.format(address))
     balance = client.get_balance(address)
-    return {'balance': balance}
+    return {'balance': balance,
+            'address': address}
```

Comment:
Why would we need this? The address is always clear, because it is always the only address provided in a request.

Reply:
For some reason, request requires a public key, not an account address.
(I didn't make that design)